### DISCUSSION TITLE
Lavarel - Développement local - Erreur

### DISCUSSION THREAD

**Discussion by etienne-64** [dbarzin/mercator#95] [Q&A] [answered]
Après l'installation en local et le déploiement des librairies (composer update, php artisan vendor:publish --all), aucun souci pour faire tourner l'app avec `artisan serve`. En revanche, j'ai une erreur avec `artisan route:list` . 

![image](https://user-images.githubusercontent.com/10651729/165777488-fa56321c-f288-4792-b652-19e4faf252d7.png)

Ai-je raté quelque chose dans la configuration ?

NB: toujours dans mes premiers pas avec Laravel, pas encore tous les réflexes.

**Comment by dbarzin** [comment, marked as answer]
Il y avait un problème avec plusieurs routes sans doutes suite à la migration vers Laravel 9.
La liste des routes refonctionne.

**Reply by etienne-64** [reply]
Confirmé.  Je peux ainsi continuer à me "battre" pour rapatrier la branche dev ;)
